### ISSUE TITLE
Some test takes over 10 minutes for 1 trip, any ideas?

### ISSUE THREAD

**Issue by tomthedude** [sitespeedio/sitespeed.io#2149]
Hello guys,
i am using ubuntu 16 vps, but running docker, system have 4 cpu and 10 ram, should be enough for just 1 test.
some websites it takes around 1.5 for a trip of collecting data (i use only 1 trip every time)
some trips take over 10 minutes even though the site actually gets loaded after 1 min tops.
it keeps running 3 instances of chrome for 1 test, also FFMPEG is taking up a lot of procesor power, even though i disabled video recording.
i am using 7.4.0
here is the command i used:
docker run --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:7.4.0 https://ynet.co.il -n 1 --plugins.add analysisstorer --video false --plugins.remove html

here is the log(1 of many, 10 minutes log...):
[2018-09-19 19:32:01] INFO: [sitespeedio] Versions OS: linux 4.4.0-112-generic nodejs: v8.11.1 sitespeed.io: 7.3.6 browsertime: 3.3.1 coach: 2.0.6
[2018-09-19 19:32:01] INFO: [sitespeedio.plugin.browsertime] Starting chrome for analysing https://ynet.co.il 1 time(s)
[2018-09-19 19:32:01] INFO: [browsertime] Testing url https://ynet.co.il iteration 1
[2018-09-19 19:42:47] INFO: [browsertime] 694 requests, 5075.25 kb, backEndTime: 150ms, firstPaint: 597ms, firstVisualChange: 667ms, DOMContentLoaded: 5.65s, Load: 127.94s, speedIndex: 3651, visualComplete85: 5.63s, lastVisualChange: 130.17s, rumSpeedIndex: 1188
[2018-09-19 19:42:49] INFO: [sitespeedio] Finished analysing https://ynet.co.il

as you can see the website itself did not take that much time to load, how come the test got spread over 10 minutes?
and other test i make with same parameters finish after 1 and a half minute give or take.
please advice.

**Comment by soulgalore** [comment]
Hey @tomthedude 
it's not obvious but to fully remove the video you need to disable getting visualMetrics too.

```
$ sitespeed.io --help
....
  --browsertime.video, --video                                   Record a video and store the video. Set it to false to
                                                                 remove the video that is created by turning on
                                                                 visualMetrics. To remove fully turn off video recordings,
                                                                 make sure to set video and visualMetrics to false.
...
```
The wording there isn't optimal, lets change that to the next release. So to fully exclude using ffmpeg add `--video false --visualMetrics false`.

In your example the last visual change is 130.17s = 2 minutes, then the video will be quite large so it takes a lot of processing. I always run one test at a time on a machine since else metrics can be affected.

Best
Peter

**Comment by tomthedude** [comment]
yeah, that was probably the the issue, here is the command with --visualmetrics false and the result (over all test time went down from 10 minutes to 2 minuts~ :) ):
:~# docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io https://ynet.co.il -n 1 --video false --visualMetrics false
Google Chrome 69.0.3497.81
Mozilla Firefox 62.0
[2018-09-20 12:11:10] INFO: Versions OS: linux 4.4.0-112-generic nodejs: v8.11.1 sitespeed.io: 7.4.0 browsertime: 3.4.1 coach: 2.1.0
[2018-09-20 12:11:12] INFO: Starting chrome for analysing https://ynet.co.il 1 time(s)
[2018-09-20 12:11:12] INFO: Testing url https://ynet.co.il iteration 1
[2018-09-20 12:13:30] INFO: 508 requests, 6084.75 kb, backEndTime: 199ms, firstPaint: 1.23s, DOMContentLoaded: 6.55s, Load: 128.64s, rumSpeedIndex: 1444
[2018-09-20 12:13:34] INFO: HTML stored in /sitespeed.io/sitespeed-result/ynet.co.il/2018-09-20-12-11-10
[2018-09-20 12:13:34] INFO: Finished analysing https://ynet.co.il
